packages/nns/src/canisters/governance/response.converters.ts

```diff
@@ -1081,6 +1085,116 @@ export const toNetworkEconomics = (
   ),
 });
 
+const toNeuronSubsetMetrics = (
+  rawNeuronSubsetMetrics: Nullable<RawNeuronSubsetMetrics>,
+): Option<NeuronSubsetMetrics> => {
+  const neuronSubsetMetrics = fromNullable(rawNeuronSubsetMetrics);
+
+  if (isNullish(neuronSubsetMetrics)) {
```

Comment:
Is there is an explicit test for this use case? If no, would that be interesting?

Reply:
Done :)